example/lib/styles/colors.dart

```diff
@@ -29,41 +29,29 @@ class AppColors {
   static const Color tertiaryAlt = Color(0xffF3CEF3);
   static const Color tertiaryAltDark = Color(0xff532D53);
 
-  static const Color success10 = Color(0x33148F47);
-  static const Color success10Dark = Color(0x1A55E792);
-
-  static const Color success20 = Color(0x33148F47);
-  static const Color success20Dark = Color(0x3355E792);
-
-  static const Color warning10 = Color(0x1AFFB938);
-  static const Color warning10Dark = Color(0x1AFFB938);
-
-  static const Color error10 = Color(0x1ACD1D32);
-  static const Color error10Dark = Color(0x1AD42136);
-
   // On Colors
-  static const Color onPrimary = Color(0xffF5F8FF);
+  static const Color onPrimary = Color(0xffF5F9FF);
   static const Color onPrimaryDark = Color(0xffFAFCFF);
```

Comment:
@kalpeshp0310 I notice that the color values for the dark theme have not changed.
I used these [design](https://www.figma.com/file/pxXcMnd1EtFLny3NQy1s8R/Tarka-kit--Dark-mode-(For-devs)?type=design&node-id=1220%3A142028&mode=design&t=bedlCVeTFqLndPzs-1) to set the color values for dark mode.

Reply:
I don't think we need to change it. The light variant of same design had value change.
I think we are good here.